From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Johannes Schindelin <johannes.schindelin@gmx.de>
Date: Thu, 30 Jul 2026 03:20:51 +0200
Subject: [PATCH 093/109] .gitattributes: enforce Unix line endings in .sh
 scripts

Bash is _really_ unhappy with Carriage Returns:

  ../../.././winsup/testsuite/cygrun.sh: 5:
  : not found
  ../../.././winsup/testsuite/cygrun.sh: 8:
  : not found
  ../../.././winsup/testsuite/cygrun.sh: 10:
  : not found
  ../../.././winsup/testsuite/cygrun.sh: 18: Syntax error: end of file unexpected (expecting "then")

Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
(cherry picked from commit af55a0450c1723f4a68de96c1f5fd1181c98a2ab)
(cherry picked from commit e01a7c9e40f038b4dfef02eb1a1802c6856ab350)
---
 .gitattributes | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitattributes b/.gitattributes
index 06d51d28a..b7e204ab2 100644
--- a/.gitattributes
+++ b/.gitattributes
@@ -18,3 +18,4 @@
 #     file gets renamed again ...
 
 ChangeLog       merge=merge-changelog
+*.sh eol=lf
